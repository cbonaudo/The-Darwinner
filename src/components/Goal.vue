<template>
  <div class="goal">
    <h2>How to play :</h2>
    <h3>Part I - The Cells Era</h3>
    <ul>
      <li>Divide Cells</li>
      <li>Buy Upgrades (cost 90% of your cells)</li>
      <li>Repeat</li>
      <li>$$-Profit-$$</li>
    </ul>
  </div>
</template>

<script>
export default {
  name: "Goal"
};
</script>

<style>
.goal {
  text-align: left;
  position: absolute;
  right: 5%;
  top: 0;
}
</style>
