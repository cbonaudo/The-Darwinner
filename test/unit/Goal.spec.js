import Goal from '@/components/Goal'
import { shallowMount } from '@vue/test-utils'

describe('Goal.vue', () => {
  const wrapper = shallowMount(Goal)

  it('should render correct contents', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('should have correct starting data', () => {
    expect(Goal.data()).toMatchSnapshot()
  })
})
