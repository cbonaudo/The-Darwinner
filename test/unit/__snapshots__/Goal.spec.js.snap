// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Goal.vue should have correct starting data 1`] = `
Object {
  "atomCeiling": 210,
}
`;

exports[`Goal.vue should render correct contents 1`] = `
<div
  class="goal"
>
  <h2>
    How to play :
  </h2>
   
  <div>
    <h3>
      Error - Era
    </h3>
     
    <p>
      Someone messed up big
    </p>
  </div>
   
  <!---->
   
  <!---->
</div>
`;
